rust/operator-binary/src/controller.rs

```diff
@@ -848,7 +804,6 @@ fn validated_product_config(
 
     let config_files = vec![
         PropertyNameKind::File(CONFIG_PROPERTIES.to_string()),
-        PropertyNameKind::File(HIVE_PROPERTIES.to_string()),
```

Comment:
Not that we currently have any checks, but this means the hive properties are not run through the product config?

Reply:
Exactly. This applies to all the catalog in the future as well. We only start with hive here, but IMHO iceberg, postgres, pheonix, druid etc. will follow

Reply:
But you could still run it through the product config? When creating the config maps you can match on the filename and add properties or alter the directory?

Reply:
I fear i don't have enough knowledge about the product-config-machinery to see whats possible and/or meaningful to validate here. Would be very happy to have a short call discussing this.

Reply:
yeah we can have a call about that